metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -728,7 +725,7 @@ def quickstart(
         )
 
     if build_locally:
-        click.echo("Building docker images locally...")
+        logger.info("Building docker images locally...")
```

Comment:
What's the benefit of using click.echo? Don't we still use it all over the place?

Reply:
logger gets the timestamps

this isn't a common option so we don't need to output to be simple